### PR TITLE
#1 Initial CircleCI support

### PR DESCRIPTION
**Pull request by stellarspot**
The provided CircleCI configuration setups all necessary tools for SingularityNET platform building.
It gets from git and build modules platform-contracts, snet-cli, snet-daemon, and example-service.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - run:
+        name: Build snet-daemon
+        command: |
+          export PATH=$PATH:$GOPATH/bin
+          rm -rf $GOPATH/bin
+          cd $SINGNET_REPOS
+          git clone https://github.com/singnet/snet-daemon.git
+          cd snet-daemon
+          ./scripts/install
+          ./scripts/build
```
Comment (vsbogd) [inline .circleci/config.yml]: Should be ```./scripts/build linux amd64```, I have fixed wiki already https://github.com/singnet/wiki/wiki/Tutorial:-Build-and-deploy-SingularityNET-locally#build-snet-daemon

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - run:
+        name: Build example-service
+        command: |
+          export PATH=$PATH:$GOPATH/bin
+          cd $SINGNET_REPOS
+          git clone https://github.com/singnet/example-service.git
+          cd example-service
+          pip3 install -r requirements.txt
```
Comment (vsbogd) [inline .circleci/config.yml]: really minor, but it is better to add newline at the end of last line

File: .circleci/config.yml (hunk 1 above)
Comment (AlexanderScherbatiy) [reply .circleci/config.yml]: I have added linux amd64 arguments for the snet-daemon building.

File: .circleci/config.yml (hunk 2 above)
Comment (AlexanderScherbatiy) [reply .circleci/config.yml]: I added the new line to the config file in the next pull request #4 which provides the integration test

**Pull request merged**